internal/rm/nvml_manager.go

```diff
@@ -112,6 +112,11 @@ func (r *nvmlResourceManager) getPreferredAllocation(available, required []strin
 // alignedAlloc shells out to the alignedAllocationPolicy that is set in
 // order to calculate the preferred allocation.
 func (r *nvmlResourceManager) alignedAlloc(available, required []string, size int) ([]string, error) {
+	// return all available directly if allocating all available devices
```

Comment:
One comment / question I have is whether this is applicable to all allocation types, meaning that we should check this in `getPreferredAllocation` or in `GetPreferredAllocation`?